### PR TITLE
🐛Don't report error when amp-google-client-id-api is set to gtag

### PR DESCRIPTION
**Pull request by zikas**

### CODE REVIEW

**Comment by lannka** [comment]
@zikas @avimehta could you explain a bit why we want to suppress the error message? Reading the comments, I feel that's user's mis-config which should be warned?

**Comment by avimehta** [comment]
@lannka We added the meta tag recommendation to gtag documentation when client id api was something that we were planning to use. We have since updated our plans. If you would like, we could add an api key for gtag but, instead, we thought hiding this warning will have the same effect. Deferring to you for what is best for the codebase.

**Comment by zikas** [comment]
There's even bigger problem that the default snippet Ads used to generate contained <meta> tag, meaning everybody who already copy-pasted generated snippet started getting error message. 
Our original plan to go with client ID api didn't work as it seems like overall plan to sunset this API.
We've updated the snippet and this is _a_ way to cope with existing installations.

**Comment by lannka** [comment]
I see.

Should we print a more relevant error message so that developers are aware that they should migrate the snippet?

**Comment by zikas** [comment]
Maybe not an error, but a warning suggesting that content="gtag" is not supported and must be removed? @avimehta , what do you think?

**Comment by avimehta** [comment]
Yes, I think that works as well.

**Comment by lannka** [comment]
@zikas are you still on this PR?

**Comment by zikas** [comment]
Got sidetracked, PTAL?

**Comment by lannka** [comment]
I meant to improve the generic error msg, not introducing a special msg for gtag.
Pseudo code :

```
Unsupported client for Google CID API: ${clientName}. Please remove or correct ${optInMeta.outerHtml}
```

It should print something like:
> Unsupported client for Google CID API: gtag. Please remove or correct `<meta name="amp-google-client-id-api" content="gtag">`

**Comment by zhouyx** [comment]
+1 to not hardcode gtag error msg

**Comment by zikas** [comment]
Updated 
https://screenshot.googleplex.com/6uUJdMbchQN.png

**Comment by zikas** [comment]
also changed to warning, please let me know if you'd like to keep it error

File: src/service/cid-impl.js [hunk 1]
```diff
@@ -305,9 +305,11 @@ export class Cid {
           if (scope) {
             apiKeyMap[scope] = API_KEYS[clientName];
           } else {
-            user().error(
+            user().warn(
               TAG_,
-              `Unsupported client for Google CID API: ${clientName}`
+              `Unsupported client for Google CID API: ${clientName}. Please remove or correct ${
```
Comment (lannka) [inline src/service/cid-impl.js]: nit, move `${optInMeta./*OK*/outerHTML}` to same line. line break before `Please remove...`

Comment (zikas) [reply src/service/cid-impl.js]: hm.. this was auto-formatted.

File: src/service/cid-impl.js [hunk 2]
```diff
@@ -305,9 +305,10 @@ export class Cid {
           if (scope) {
             apiKeyMap[scope] = API_KEYS[clientName];
           } else {
-            user().error(
+            user().warn(
               TAG_,
-              `Unsupported client for Google CID API: ${clientName}`
+              `Unsupported client for Google CID API: ${clientName}. 
+Please remove or correct ${optInMeta./*OK*/ outerHTML}`
```
Comment (rsimha) [inline src/service/cid-impl.js]: Drive-by review: If you want to avoid the `\n` in the warning string, you can rewrite this as:
```js
            user().warn(
              TAG_,
              `Unsupported client for Google CID API: ${clientName}. ` +
                `Please remove or correct ${optInMeta./*OK*/ outerHTML}`
            );
```

Comment (zikas) [reply src/service/cid-impl.js]: sure, updated
